corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -484,17 +381,126 @@ def _utcnow():
     return datetime.utcnow()
 
 
-def _format_last_sync_time(domain, user_id, sync_time=None):
+def _format_last_sync_time(restore_user, sync_time=None):
     sync_time = sync_time or _utcnow()
-    timezone = get_timezone_for_user(user_id, domain)
+    timezone = get_timezone_for_user(restore_user._couch_user, restore_user.domain)
     return ServerTime(sync_time).user_time(timezone).done().isoformat()
 
 
-report_fixture_generator = ReportFixturesProvider()
-report_fixture_v2_generator = ReportFixturesProviderV2()
+def generate_rows_and_filters(report_data_cache, report_config, restore_user, row_to_element):
+    """Generate restore row and filter elements
+    :param row_to_element: function (
+                deferred_fields, filter_options_by_field, row, index, is_total_row
+            ) -> row_element
+    """
+    domain = restore_user.domain
+    report, data_source = _get_report_and_data_source(report_config.report_id, domain)
```

Comment:
Will these not get looked up twice for v1/v2 since `generate_rows_and_filters` would get called from v1/v2?

Reply:
could do I guess, will do a follow up PR